node/src/BaseClient.ts

```diff
@@ -2411,6 +2418,7 @@ export class BaseClient {
      *
      * @param key - The key of the list.
      * @param count - The count of the elements to pop from the list.
+     * @param decoder - (Optional) {@link Decoder} type which defines how to handle the response. If not set, the default decoder from the client config will be used.
```

Comment:
```suggestion
     * @param decoder - (Optional) {@link Decoder} type which defines how to handle the response.
     *     If not set, the {@link BaseClientConfiguration.defaultDecoder|default decoder} will be used.
```

Reply:
Please update it in all places including already merged onces.

Reply:
Updated in all relevant places